src/pudl/io_managers.py

```diff
@@ -535,10 +503,29 @@ def load_input(self, context: InputContext) -> pd.DataFrame:
         return df
 
 
-@io_manager
-def pudl_sqlite_io_manager(init_context) -> PudlSQLiteIOManager:
+@io_manager(
+    config_schema={
+        "write_to_parquet": Field(
+            bool,
+            description="""If true, data will be written to parquet files,
+                in addition to the SQLite database.""",
+            default_value=bool(os.getenv("PUDL_WRITE_TO_PARQUET")),
+        ),
+        "read_from_parquet": Field(
+            bool,
+            description="""If True, the canonical source of data for reads
+                will be parquet files. Otherwise, data will be read from the
+                SQLite database.""",
+            default_value=bool(os.getenv("PUDL_READ_FROM_PARQUET")),
+        ),
+    }
+)
+def pudl_io_manager(init_context) -> IOManager:
```

Comment:
Agreed! I think the resource key for assets should be `pudl_io_manager` but the IO managers should have more specific names like `pudl_mixed_format_io_manager` the type of IO manager the `pudl_io_manager` key refers to.

Reply:
Wouldn't this be achieved via migrating to the new dagster configuration/resources model where we just instantiate whatever iomanager we want under `pudl_io_manager` key?

That said, I do not expect that we will really want to swap out iomanagers here that often if at all. I anticipate that we want pretty uniform behavior and pretty fixed formats. Adding support for dual writes/reads is one case where this is getting violated, but I'm unsure if we really need to be planning on too much flexibility here just yet.

Reply:
I spent a long time confused about where the IO Manager was coming from because the same name was used in every context, so just for legibility I'd prefer that we keep the names distinct here. I agree we don't anticipate swapping in lots of different IO Managers in different test/prod environments, but given that the change is tiny and also helps with legibility I see no reason not to do it.